sim/team-validator.js

```diff
@@ -921,22 +921,6 @@ class Validator {
 		while (template && template.species && !alreadyChecked[template.speciesid]) {
 			alreadyChecked[template.speciesid] = true;
 			if (dex.gen === 2 && template.gen === 1) tradebackEligible = true;
-			// STABmons hack to avoid copying all of validateSet to formats
-			// @ts-ignore
-			let noLearn = format.noLearn || [];
-			if (ruleTable.has('ignorestabmoves') && !noLearn.includes(move.name) && !move.isZ) {
```

Comment:
I still think that this should be kept as an independent check so that `Ignore STAB Moves` can still be applied onto tournaments.

Reply:
I'd probably give it a better name, though.

Something like `STABmons Move Legality`.

Reply:
yeah